src/client/containers/404Page/404Page.component.js

```diff
@@ -1,4 +1,30 @@
 import React from 'react';
 import HeaderTitle from '../../components/Title/Title';
+import ButtonComponent from '../../components/SubmitButton/SubmitButton.js';
+import FooterComponent from '../../components/FooterComponent/FooterComponent.js';
+import ErrorLogo from '../../assets/images/background/flag404.png';
+import '../../App.css';
+import './404Page.styles.css';
 
-export const ErrorPage = () => <HeaderTitle title="This is the 404 Page" />;
+export const ErrorPage = () => {
+  return (
+    <div className="bodycontainer error-page-container">
+      <HeaderTitle title="Error 404" />
+      <div className="bodywrapper">
+        <div className="error-logo-container">
+          <img src={ErrorLogo} alt="Error Logo" />
+          <h3>Oh no!</h3>
+          <p>
+            We are usually a treasure chest, but we could not find what you are
+            looking for.
+          </p>
+        </div>
+        <ButtonComponent
+          ButtonTitle="Try again"
+          onClick={() => history.go(-1)}
```

Comment:
Isn't `history` here `undefined`?

Reply:
@bytewiz  No. its not undefined. history is web API. It contains the URLs visited by the user (within a browser window).  https://developer.mozilla.org/en-US/docs/Web/API/History_API

Reply:
I see, but there is no reference to the window object. Also if you just have is the idea that the button "refreshes" the page? then I think this way would be better: https://www.w3schools.com/jsref/met_loc_reload.asp

Reply:
yeah thanks :) I am going to add window as reference now. Yes i had the idea of page refreshing, but then it just refreshes and shows the same error page.

Reply:
Makes sense 👍